dspace/config/dspace.cfg

```diff
@@ -1624,6 +1624,33 @@ google.recaptcha.site-verify = https://www.google.com/recaptcha/api/siteverify
 # checkbox - The "I'm not a robot" Checkbox requires the user to click a checkbox indicating the user is not a robot.
 #google.recaptcha.mode =
 
+#------------------------------------------------------------------#
+#---------------REGISTRATION DATA CONFIGURATION--------------------#
+#------------------------------------------------------------------#
+
+# Configuration for the duration of the token depending on the type
+# the format used should be compatible with the standard DURATION format,
+# but without the prefix `PT`:
+#
+# - PT1H -> 1H // hours
+# - PT1M -> 1M // minutes
+# - PT1S -> 1S // seconds
+#
+eperson.registration-data.token.orcid.expiration = 1H
+eperson.registration-data.token.validation_orcid.expiration = 1H
+eperson.registration-data.token.forgot.expiration = 24H
+eperson.registration-data.token.register.expiration = 24H
+eperson.registration-data.token.invitation.expiration = 24H
+eperson.registration-data.token.change_password.expiration = 1H
```

Comment:
Each of these configs needs an explanation, as they are not self-explanatory. What is the difference between `orcid.expiration` and `validation_orcid.expiration`?  Please add a description to each of these configs.